src/types/AttributeOrAttributeValueFilter.ts

```diff
@@ -1,4 +1,4 @@
-export type AttributeKey = string & {readonly ___attributeKey: unique symbol};
+export type AttributeKey = string;
```

Comment:
what was the problem with this?

Reply:
I came across some GitHub issues and SO posts about similar use cases, https://stackoverflow.com/questions/58779171/typescipt-intersection-types-unique-symbol, and I thought those might be old and irrelevant, but I still get the error.

because of the `{readonly ___attributeKey: unique symbol}` we have to explicitly cast the attribute value
```
Type 'string' is not assignable to type 'AttributeKey'.
```

![Screenshot 2023-08-09 at 16 56 05](https://github.com/scayle/storefront-api-ts-sdk/assets/82959233/a115a12b-0385-45c2-9cc6-0fce1e6bb869)

maybe there was a specific use case for this to be added at some point.

Reply:
👍  fine with me to remove it then